README.md

```diff
@@ -4,13 +4,248 @@ NATS Streaming is an extremely performant, lightweight reliable streaming platfo
 
 [![License MIT](https://img.shields.io/npm/l/express.svg)](http://opensource.org/licenses/MIT) 
 [![Build Status](https://travis-ci.com/nats-io/nats-streaming-server.svg?token=UGjrGa8sFWGQcHSJeAvp&branch=master)](http://travis-ci.com/nats-io/nats-streaming-server)
-[![Coverage Status](https://coveralls.io/repos/github/nats-io/stan-server/badge.svg?branch=master&t=7dcKvx)](https://coveralls.io/github/nats-io/stan-server?branch=master)
-
-[Project Design Document](https://docs.google.com/document/d/1keDwK35YQnOXXKKy2HVV2oOnvEUPFyypT-Tplh8F89c/edit)
+[![Coverage Status](https://coveralls.io/repos/github/nats-io/nats-streaming-server/badge.svg?branch=master&t=7dcKvx)](https://coveralls.io/github/nats-io/nats-streaming-server?branch=master)
 
 NATS Streaming provides the following high-level feature set.
 - Log based.
 - At-Least-Once Delivery model, giving reliable message delivery.
 - Rate matched on a per subscription basis.
 - Replay/Restart
 - Last Value Semantics
+
+### Getting Started
+
+The best way to get the NATS Streaming Server is to use one of the pre-built release binaries which are available for OSX, Linux (x86-64/ARM), Windows. Instructions for using these binaries are on the GitHub releases page.
+
+Of course you can build the latest version of the server from the master branch. The master branch will always build and pass tests, but may not work correctly in your environment. You will first need Go installed on your machine (version 1.5+ is required) to build the NATS server.
+
```

Comment:
Add another para to the end. "See also the NATS Streaming Quickstart tutorial." with the following link: http://site-nats-streaming-staging.zeppole.buffalo.im/documentation/tutorials/nats-streaming-quickstart/.